appveyor.yml

```diff
@@ -1,31 +1,49 @@
-image: Visual Studio 2017
+image: Visual Studio 2019
 clone_folder: c:\projects\darknet
 cache: C:\Tools\vcpkg\installed\
 
 environment:
     WORKSPACE: C:\projects
     matrix:
     - platform: Cygwin64
+      USE_CUDA: no
       COMPILER: cygwin
       CYGWIN_NOWINPATH: yes
       CYGSH: C:\cygwin64\bin\bash -c
     - platform: Win64
       USE_CUDA: yes
+      INSTALL_CUDA_VERSION: 10.0
       COMPILER: vs
       configuration: Release
       VCPKG_ROOT: C:\Tools\vcpkg
       VCPKG_DEFAULT_TRIPLET: x64-windows
+    - platform: Win64
+      USE_CUDA: yes
+      INSTALL_CUDA_VERSION: 10.1
+      COMPILER: vs
+      configuration: Release
+      VCPKG_ROOT: C:\Tools\vcpkg
+      VCPKG_DEFAULT_TRIPLET: x64-windows
+    #- platform: Win64
+    #  USE_CUDA: yes
+    #  INSTALL_CUDA_VERSION: 10.2
+    #  COMPILER: vs
+    #  configuration: Release
+    #  VCPKG_ROOT: C:\Tools\vcpkg
+    #  VCPKG_DEFAULT_TRIPLET: x64-windows
     - platform: Win64
       USE_CUDA: no
       COMPILER: vs
       configuration: Release
       VCPKG_ROOT: C:\Tools\vcpkg
       VCPKG_DEFAULT_TRIPLET: x64-windows
     - platform: Win64
+      USE_CUDA: no
```

Comment:
Why did you add USE_CUDA: no there?

Reply:
to test configurations with and without Cuda